benchalerts/benchalerts/pipeline_steps/conbench.py

```diff
@@ -11,17 +12,46 @@
 ConbenchClient = LegacyConbenchClient
 
 
-class GetConbenchZComparisonStep(AlertPipelineStep):
-    """An ``AlertPipeline`` step to get information from Conbench comparing the runs on
-    a contender commit to their baselines, using a z-score threshold. This is always the
-    first step of the pipeline.
+CONBENCH_ENV_VAR_HELP = """
+
+    Notes
+    -----
+    Environment variables
+    ~~~~~~~~~~~~~~~~~~~~~
+    ``CONBENCH_URL``
+        The URL of the Conbench server. Only required if ``conbench_client`` is not
+        provided.
+    ``CONBENCH_EMAIL``
+        The email to use for Conbench login. Only required if ``conbench_client`` is not
+        provided and the server is private.
+    ``CONBENCH_PASSWORD``
+        The password to use for Conbench login. Only required if ``conbench_client`` is
+        not provided and the server is private.
+"""
```

Comment:
Realized later the github steps just duplicate the equivalent section. Opinions on how DRY we want our docstrings?

Reply:
See my other comment